tests/unit/core/test_addonManager.js

```diff
@@ -238,41 +256,116 @@ describe(__filename, () => {
       addEventListener: sinon.stub(),
     };
 
-    const handleChangeEvent = addonManager.addChangeListeners(
-      fakeEventCallback,
-      fakeMozAddonManager,
-    );
+    beforeEach(() => {
+      fakeEventCallback.resetHistory();
+      fakeMozAddonManager.addEventListener.resetHistory();
```

Comment:
I took a closer look and you don't need to redefine `fakeMozAddonManager` here at all. You can just delete it. The current definition looks right:

https://github.com/mozilla/addons-frontend/blob/eb1134bf3a4f00e7f49be92aee7532aca8e41189/tests/unit/core/test_addonManager.js#L42-L46